### PR TITLE
Add page.codeberg.tpikonen.satellite

### PR DESCRIPTION
**Pull request by tpikonen**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project.
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: page.codeberg.tpikonen.satellite.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (barthalion) [inline page.codeberg.tpikonen.satellite.appdata.xml:1]: Ideally, you should maintain this file in your upstream repo. It's not specific to Flathub, and you can also update it when making a new release.

File: page.codeberg.tpikonen.satellite.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  "copy-icon": false,
+  "rename-desktop-file": "satellite.desktop",
+  "finish-args": [
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--share=ipc",
+    "--device=dri",
+    "--talk-name=org.gtk.vfs.*",
+    "--system-talk-name=org.freedesktop.ModemManager1.*",
+    "--filesystem=home"
```
Comment (barthalion) [inline page.codeberg.tpikonen.satellite.json:17]: What does it use `home` for?

File: page.codeberg.tpikonen.satellite.appdata.xml (hunk 1 above)
Comment (tpikonen) [reply page.codeberg.tpikonen.satellite.appdata.xml]: I removed the appdata, it's indeed already in the sources.

File: page.codeberg.tpikonen.satellite.json (hunk 2 above)
Comment (tpikonen) [reply page.codeberg.tpikonen.satellite.json:17]: For saving GPX-tracks.

**Comment by tpikonen** [comment]
bot, build page.codeberg.tpikonen.satellite

File: page.codeberg.tpikonen.satellite.json (hunk 2 above)
Comment (barthalion) [reply page.codeberg.tpikonen.satellite.json:17]: Ideally, it should use GtkNativeFileChooser to use a portal instead of direct filesystem access.

Comment (tpikonen) [reply page.codeberg.tpikonen.satellite.json:17]: This app does not use a file chooser, but writes tracks to a file with a name generated from a timestamp (like `track-2021-09-06_14:09:22.gpx`).

It currently only writes them to G_USER_DIRECTORY_DOCUMENTS (i.e. `~/Documents`), but I can further restrict this to a subdir like `satellite-tracks`. Then the needed filesystem permissions would be `--filesystem=xdg-documents/satellite-tracks:create` (if I'm reading the [docs](https://docs.flatpak.org/en/latest/sandbox-permissions-reference.html#filesystem-permissions) correctly).

Would this be ok?

Comment (barthalion) [reply page.codeberg.tpikonen.satellite.json:17]: Yeah, that sounds fine.

**Comment by tpikonen** [comment]
bot, build page.codeberg.tpikonen.satellite

File: page.codeberg.tpikonen.satellite.json (hunk 2 above)
Comment (tpikonen) [reply page.codeberg.tpikonen.satellite.json]: Ok, I made this change on a new release and updated the build manifest.

**Pull request closed without merging**